### ISSUE TITLE
train loss is negative

### ISSUE THREAD

**Issue by XiaoYangon** [JiahuiYu/generative_inpainting#372]
hello，thanks for  release deepfillv2. I have loade code and pretrained models. I want to train my dataset(256*256), so I use pretrained models to fine-tune train, I changes some trianing parameters，but the train loss is negative and decreasing，I suspect it's my own data problem. so I use place2 validation set to fine-tune train，the train loss is also negative，I don't know why ，can you tell me，Is code error？thanks

**Comment by JiahuiYu** [comment]
#371
